entity-utils/src/main/java/eu/europeana/entity/utils/EntityUtils.java

```diff
@@ -48,4 +48,21 @@ public static List<String> updateBaseUrlInIds(List<String> entityUris, String da
         }
         return updatedUris;
     }
+    
+    public static String validateUriAndRemoveQuotes(String uri) {
+        boolean startsWithDoubleQuotes = uri.startsWith("\"");
+        boolean endsWithDoubleQuotes = uri.endsWith("\"");
+        
+        if(startsWithDoubleQuotes && endsWithDoubleQuotes) {
+          if(UriValidator.isUri(uri.substring(1, uri.length()-1))) {
+            return uri.substring(1, uri.length()-1);
+          }
+        }
+        if(!startsWithDoubleQuotes && !endsWithDoubleQuotes) {
```

Comment:
the URI validation must be allways performed, independent if the provided URL contains or not trailing quotes